rest-helix/src/main/java/com/github/twitch4j/helix/TwitchHelixErrorDecoder.java

```diff
@@ -88,7 +88,7 @@ public Exception decode(String methodKey, Response response) {
                 // If you get an HTTP 503 (Service Unavailable) error, retry once.
                 // If that retry also results in an HTTP 503, there probably is something wrong with the downstream service.
                 // Check the status page for relevant updates.
-                ex = new RetryableException(response.status(), "getting service unavailable, retrying ...", Request.HttpMethod.GET, null, response.request());
+                ex = new RetryableException(response.status(), "getting service unavailable, retrying ...", Request.HttpMethod.GET, (Long) null, response.request());
```

Comment:
Alternatively we could cast to `Date` (targets a deprecated constructor), which would maintain compatibility with v12 (if we're concerned about feign being declared as `api` instead of `implementation`)